web/e2e/page-objects/flows/editFlow.ts

```diff
@@ -148,6 +148,10 @@ export class EditFlow extends AppPage {
     return await menuOption.click();
   }
 
+  get stepDialogBox() {
+    return element(by.css("mat-dialog-container"));
```

Comment:
is this supposed to be element or class? look at flowDialogBox below

Reply:
Both work "<mat-dialog-container aria-modal="true" class="mat-dialog-container ng-tns-c25-10 ng-trigger ng-trigger-dialogContainer ng-star-inserted" tabindex="-1""